taxi/forms.py

```diff
@@ -0,0 +1,60 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Manufacturer, Car
+
+
+class DriverForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number", "first_name", "last_name")
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) == 8 and license_number[:3].isupper() and license_number[3:].isdigit():
+            return license_number
+        raise ValidationError(
+            "Ensure that value is incorrect: "
+            "* Consist only of 8 characters "
+            "* First 3 characters are uppercase letters "
+            "* Last 5 characters are digits "
+        )
+
+
+class DriverUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) == 8 and license_number[:3].isupper() and license_number[3:].isdigit():
+            return license_number
+        raise ValidationError(
+            "Ensure that value is incorrect: "
+            "* Consist only of 8 characters "
+            "* First 3 characters are uppercase letters "
+            "* Last 5 characters are digits "
+        )
```

Comment:
code duplication. Take this function outside class and use it in all forms where needed